ml-agents/mlagents/trainers/trainer_controller.py

```diff
@@ -181,12 +184,29 @@ def _save_model(self,steps=0):
             self.trainers[brain_name].save_model()
         self.logger.info('Saved Model')
 
+    def _win_handler(self, event):
+        """
+        This function gets triggered after ctrl-c or ctrl-break is pressed 
+        under Windows platform.
+        """
+        if event in (win32con.CTRL_C_EVENT, win32con.CTRL_BREAK_EVENT):        
+            print('--------------------------Now saving model--------------'
+                      '-----------')
+            self.logger.info('Learning was interrupted. Please wait '
+                                     'while the graph is generated.')
+            self._save_model(self.global_step)
+            self._export_graph()
+            sys.exit()
+            return True
+        return False
+
     def _export_graph(self):
         """
         Exports latest saved models to .bytes format for Unity embedding.
         """
         for brain_name in self.trainers.keys():
             self.trainers[brain_name].export_model()
+        self.logger.info("Exported .bytes file")
```

Comment:
Should this be `self.logger.debug`?

Reply:
I used self.logger.info here because this is to notify our user the .bytes file is exported, similar to what we did in def _save_model(self,steps=0):
where we said `self.logger.info('Saved Model')`

Reply:
Fixed